Azure DevOps stoped working on Rider 2022.2

Plugin doesn't work anymore

```
java.lang.NoClassDefFoundError: com/intellij/openapi/vcs/changes/ui/ChangesBrowser
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestDialog.createCenterPanel(CreatePullRequestDialog.java:37)
	at com.intellij.openapi.ui.DialogWrapper.init(DialogWrapper.java:1309)
	at com.microsoft.alm.plugin.idea.common.ui.common.BaseDialogImpl.<init>(BaseDialogImpl.java:59)
	at com.microsoft.alm.plugin.idea.common.ui.common.BaseDialogImpl.<init>(BaseDialogImpl.java:40)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestDialog.<init>(CreatePullRequestDialog.java:28)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestController.<init>(CreatePullRequestController.java:35)
	at com.microsoft.alm.plugin.idea.git.actions.CreatePullRequestAction.actionPerformed(CreatePullRequestAction.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:186)
	at com.intellij.openapi.actionSystem.impl.ActionButton.lambda$performAction$0(ActionButton.java:162)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:272)
	at com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:162)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:468)
	at java.desktop/java.awt.Component.processEvent(Component.java:6413)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2802)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:754)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:752)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:751)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:898)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:820)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:743)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:439)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:438)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:604)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:436)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:484)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.vcs.changes.ui.ChangesBrowser PluginClassLoader(plugin=PluginDescriptor(name=Azure DevOps, id=com.microsoft.vso.idea, descriptorPath=plugin.xml, path=~\AppData\Local\JetBrains\Toolbox\apps\Rider\ch-0\222.2964.37.plugins\com.microsoft.vso.idea, version=1.162.0, package=null, isBundled=false), packagePrefix=null, instanceId=87, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:209)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 51 more
```

Hi, 
I encounter same problem. Please fix it. PullRequests direct from Rider is a great feature!

java.lang.NoClassDefFoundError: com/intellij/openapi/vcs/changes/ui/ChangesBrowser
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestDialog.createCenterPanel(CreatePullRequestDialog.java:37)
	at com.intellij.openapi.ui.DialogWrapper.init(DialogWrapper.java:1308)
	at com.microsoft.alm.plugin.idea.common.ui.common.BaseDialogImpl.<init>(BaseDialogImpl.java:59)
	at com.microsoft.alm.plugin.idea.common.ui.common.BaseDialogImpl.<init>(BaseDialogImpl.java:40)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestDialog.<init>(CreatePullRequestDialog.java:28)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestController.<init>(CreatePullRequestController.java:35)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.VcsPullRequestsModel.createNewItem(VcsPullRequestsModel.java:184)
	at com.microsoft.alm.plugin.idea.common.ui.common.tabs.TabControllerImpl.performAction(TabControllerImpl.java:95)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.VcsPullRequestsController.performAction(VcsPullRequestsController.java:34)
	at com.microsoft.alm.plugin.idea.common.ui.common.tabs.TabControllerImpl.actionPerformed(TabControllerImpl.java:84)
	at com.microsoft.alm.plugin.idea.common.ui.common.ActionListenerContainer.triggerEvent(ActionListenerContainer.java:15)
	at com.microsoft.alm.plugin.idea.common.ui.common.ActionListenerContainer.triggerEvent(ActionListenerContainer.java:20)
	at com.microsoft.alm.plugin.idea.common.ui.common.tabs.TabFormImpl$1.actionPerformed(TabFormImpl.java:180)
	at com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:190)
	at com.intellij.openapi.actionSystem.impl.ActionButton.lambda$performAction$0(ActionButton.java:166)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:337)
	at com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:166)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:465)
	at java.desktop/java.awt.Component.processEvent(Component.java:6413)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2802)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:754)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:752)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:751)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:898)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:820)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:743)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:439)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:438)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:604)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:436)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:484)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.vcs.changes.ui.ChangesBrowser PluginClassLoader(plugin=PluginDescriptor(name=Azure DevOps, id=com.microsoft.vso.idea, descriptorPath=plugin.xml, path=~\AppData\Local\JetBrains\Toolbox\apps\Rider\ch-1\222.3345.139.plugins\com.microsoft.vso.idea, version=1.162.0, package=null, isBundled=false), packagePrefix=null, instanceId=85, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:214)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 57 more

Same issue

```
java.lang.NoClassDefFoundError: com/intellij/openapi/vcs/changes/ui/ChangesBrowser
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestDialog.createCenterPanel(CreatePullRequestDialog.java:37)
	at com.intellij.openapi.ui.DialogWrapper.init(DialogWrapper.java:1308)
	at com.microsoft.alm.plugin.idea.common.ui.common.BaseDialogImpl.<init>(BaseDialogImpl.java:59)
	at com.microsoft.alm.plugin.idea.common.ui.common.BaseDialogImpl.<init>(BaseDialogImpl.java:40)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestDialog.<init>(CreatePullRequestDialog.java:28)
	at com.microsoft.alm.plugin.idea.git.ui.pullrequest.CreatePullRequestController.<init>(CreatePullRequestController.java:35)
	at com.microsoft.alm.plugin.idea.git.actions.CreatePullRequestAction.actionPerformed(CreatePullRequestAction.java:51)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:315)
	at com.intellij.ide.actions.GotoActionAction.lambda$performActionImpl$4(GotoActionAction.java:100)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:337)
	at com.intellij.ide.actions.GotoActionAction.performActionImpl(GotoActionAction.java:99)
	at com.intellij.ide.actions.GotoActionAction.lambda$performAction$2(GotoActionAction.java:79)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:513)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:75)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:118)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:42)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:918)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:450)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:449)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:624)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:447)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ClassNotFoundException: com.intellij.openapi.vcs.changes.ui.ChangesBrowser PluginClassLoader(plugin=PluginDescriptor(name=Azure DevOps, id=com.microsoft.vso.idea, descriptorPath=plugin.xml, path=~/.local/share/JetBrains/Toolbox/apps/Rider/ch-0/222.3739.37.plugins/com.microsoft.vso.idea, version=1.162.0, package=null, isBundled=false), packagePrefix=null, instanceId=89, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:214)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 43 more

```

--- a/plugin/src/com/microsoft/alm/plugin/idea/git/ui/pullrequest/CreatePullRequestForm.java
+++ b/plugin/src/com/microsoft/alm/plugin/idea/git/ui/pullrequest/CreatePullRequestForm.java
@@ -6,7 +6,7 @@ package com.microsoft.alm.plugin.idea.git.ui.pullrequest;
 import com.intellij.icons.AllIcons;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.vcs.changes.Change;
-import com.intellij.openapi.vcs.changes.ui.ChangesBrowser;
+import com.intellij.openapi.vcs.changes.ui.SimpleChangesBrowser;
 import com.intellij.uiDesigner.core.GridConstraints;
 import com.intellij.uiDesigner.core.GridLayoutManager;
 import com.intellij.util.ui.JBUI;
@@ -214,8 +214,8 @@ public class CreatePullRequestForm implements BasicForm {
 
     private JComponent createDiffPaneBrowser(final Project project, final GitCommitCompareInfo compareInfo) {
         List<Change> diff = compareInfo.getTotalDiff();
-        final ChangesBrowser changesBrowser = new ChangesBrowser(project, null, diff, null, false, true,
-                null, ChangesBrowser.MyUseCase.COMMITTED_CHANGES, null);
+        final SimpleChangesBrowser changesBrowser = new  SimpleChangesBrowser(project,diff);
+
         changesBrowser.setChangesToDisplay(diff);
         return changesBrowser;
     }

